src/v2/models/AppState.js

```diff
@@ -24,27 +27,26 @@ export default Model.extend({
     currentFormName: 'string',
     idx: 'object',
     remediations: 'array',
-    __previousResponse: 'object'
   },
 
   derived: {
     authenticatorProfile: {
       // While we're moving toward `authenticator` platform, but still
       // need to support `factor` for certain period.
       // Could remove `factor` after it's deprecated completely.
-      deps: ['currentAuthenticator', 'authenticatorEnrollment', 'factor'],
-      fn (currentAuthenticator = {}, authenticatorEnrollment = {}, factor = {}) {
+      deps: ['currentAuthenticator', 'enrollmentAuthenticator', 'factor'],
```

Comment:
instead of `enrollmentAuthenticator` use `enrollAuthenticator`.. `enrollment` is a verb..

Reply:
`enrollmentAuthenticator` is from api contractor. might be able to clean up when we see the final JSON ( when integrating with api )